lib/commons/aria/index.js

```diff
@@ -78,16 +85,20 @@ lookupTable.attributes = {
 	},
 	'aria-invalid': {
 		type: 'nmtoken',
+		allowEmpty: true,
 		values: ['true', 'false', 'spelling', 'grammar']
 	},
 	'aria-keyshortcuts': {
-		type: 'string'
+		type: 'string',
+		allowEmpty: true
 	},
 	'aria-label': {
-		type: 'string'
+		type: 'string',
+		allowEmpty: true
 	},
 	'aria-labelledby': {
-		type: 'idrefs'
+		type: 'idrefs',
+		allowEmpty: true
```

Comment:
There is no current test to test the empty case for `aria-labelledby` - we should add one